packages/engine/paima-funnel/src/funnels/carp/funnel.ts

```diff
@@ -422,7 +436,8 @@ async function readDataInternal(
   );
 
   let grouped = groupCdeData(
-    Network.EVM,
+    chainName,
+    ConfigNetworkType.CARDANO,
```

Comment:
This is a good question. I need to re-check actually.

I guess it was `EVM` because it's part of the sync, and not the presync. The thing is, it doesn't really change anything. This function returns `PresyncChainData`, and the network type field is used for the presync processing function, but when merging this with the `ChainData` from the evm chain during the sync it's discarded. The same thing happens with the `chainName` actually...

Maybe these fields should be optional? Or maybe `composeChainData` shouldn't actually take `PresyncChainData[]`, but a more specific type.